src/appleseed.shaders/include/appleseed/maya/as_maya_helpers.h

```diff
@@ -44,43 +44,49 @@
         string as_maya_attribute_name = "defaultColor",                     \
         int as_maya_attribute_hidden = 1,                                   \
         string label = "Default Color",                                     \
-        string page = "Color Balance"                                       \
+        string page = "Color Balance",                                      \
+        int gafferNoduleLayoutVisible = 0                                   \
```

Comment:
You did mean *Nodule* and not *Module* here right? Just checking :)

Reply:
Yes, it's Nodule

Reply:
@dictoon yeah i know Franz... i don't like the mental image (no pun intended)[ it brings](https://www.merriam-webster.com/dictionary/nodule) either

Reply:
Please, don't be me, don't type this on Google Image...